app/controllers/register_controller.rb

```diff
@@ -9,8 +9,10 @@ def index
     @changes = Change.joins("LEFT OUTER JOIN statuses on statuses.change_id = changes.id")
                      .where("register_name = '#{params[:register]}' AND statuses.status = 'pending'")
 
-    @register = get_register(params[:register])._all_records
-    @register[0].try(:name) ? @register = @register.sort_by(&:name) : @register = @register.sort_by(&:key)
+
+    @register = get_register(params[:register])
+                  ._all_records
+                  .sort_by(&:key)
```

Comment:
judging by this comment https://github.com/openregister/managing-registers/issues/132#issuecomment-340467627 we should be sorting by entry timestamp instead.

Reply:
hmm not sure about entry timestamp, Blanca said to me we should so name by default and then key if no name is present...

Reply:
OK, I spoke to @elliecraven she is happy with key.